### DISCUSSION TITLE
Controlled Input with Box, Stack

### DISCUSSION THREAD

**Discussion by vitran12** [chakra-ui/panda#2507] [Q&A]
Hi everyone

**What happened**:
When using the `Input` component as a Controlled input the value changes, the cursor unfocuses from the input field.

**What you expected to happen**:
The cursor should remain focused on the input field even when the value changes.

**How to reproduce it (as minimally and precisely as possible)**:
```tsx
import { useState } from "react";
import { Input, Label, Stack } from "@/component/ui";

function App() {
  const [data, setData] = useState("");
  return (
    <Stack w="300" p="4">
      <Label htmlFor="title">Title</Label>
      <Input onChange={(e) => setData(e.target.value)} value={data} />
    </Stack>
  );
}

export default App;
```

**Anything else we need to know?**:

https://github.com/yorkie-team/yorkie-ui/assets/52884648/456c49dc-b3e0-4c93-b9fa-b23989927193

**Comment by kiruushaaa** [comment]
It seems there is something wrong with your ui-kit components implementation since the code snippet below works as expected `(0.37.2)`
```typescript
import { useState } from 'react';
import { Stack } from 'styled-system/jsx';

export const App = () => {
  const [val, setVal] = useState('');

  return (
    <>
      <Stack>
        <label htmlFor="test-id">test</label>
        <input type="text" id="test-id" value={val} onChange={e => setVal(e.target.value)}/>
      </Stack>
    </>
  );
};
```
